### PR TITLE
Adding comments related to troubleshooting low quality output

### PR DESCRIPTION
**Pull request by CoryCharlton**
## Description
- Adding comments related to troubleshooting low quality output
- Fixed issue with I2S bus 2 not working

## Motivation and Context
- Save the next use time in troubleshooting

## How Has This Been Tested?
- On Device

## Types of changes

- [X] Improvement (non-breaking change that improves a sample)
- [X] Bug fix (fixes an issue with a current sample)
- [ ] New Sample (adds a new sample)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by Ellerbach — APPROVED** [review]
Thanks for the improvement!

File: samples/I2S/Output/I2sWavPlayer.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ namespace AudioPlayer
     /// You have to provide pin configuration for I2S communication and a full path to the
     /// WAV file you want to play.
     /// </summary>
-    public class I2sWavPlayer : IDisposable
+    public class I2SWavPlayer : IDisposable
```
Comment (josesimoes) [inline samples/I2S/Output/I2sWavPlayer.cs:19]: Why the name change? The casing rules call for `I2s` as it was, not `I2S`...

Comment (CoryCharlton) [reply samples/I2S/Output/I2sWavPlayer.cs:19]: https://github.com/nanoframework/Samples/pull/327

Comment (josesimoes) [reply samples/I2S/Output/I2sWavPlayer.cs:19]: No worries! This is nothing major, just trying to have things consistent and tidy. 😉

**Pull request merged**